src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,14 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040000
+name: GBA1-related Parkinson disease
+def: "Increased susceptibility for developing Parkinson disease due to the presence of a monoallelic pathogenic variant, likely pathogenic variant, or risk allele in the GBA1 gene." [https://orcid.org/0000-0003-3254-7564, PMID:19433656, PMID:19846850, PMID:19858467]
```

Comment:
please add the ClinGen working group as x-ref.

Reply:
Comment addressed.